source/security.rst

```diff
@@ -0,0 +1,38 @@
+.. _security:
+
+Security
+=================
+
+Introduction
+------------
+
+This document provides an overview of the security framework implemented in Open OnDemand, detailing security features, general policies, and procedures.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Security Topics
+
+   security/vulnerability-management
+   authentication/overview 
+   how-tos/monitoring/logging
+   customizations
+
+Security Features
+-----------------
```

Comment:
I think the main point here should be more like `Security Concerns`. Here are some off bullet points I'd add/think folks are expecting.

I think folks are trying to answer the question `if I install this, what security concerns do I need to account for?`.  This page should answer that.

The good:

* PUN (Per user Nginx) architecture. The web servers that the user will ultimately interact with use are processes' ran by that user. That is, the `root` user does not serve requests. Only non-root users serve requests and user request are always routed to the nginx of that same user. This ensures that things like file accesses (or any action taken by the webservers) are always made under the non-root user.
* Apache authentication is required. The type of authentication scheme is up to the individual site. Basic authentication mechanisms like Basic or LDAP are discouraged and undocumented because they are insecure. 

The bad:
* HTTP only traffic to origin servers (compute nodes). When running applications like Jupyter, apache will proxy traffic back to the origin server (Jupyter on a compute node) over plain http traffic.  There's ongoing work to make this https traffic.